app/models/thredded/null_user.rb

```diff
@@ -34,6 +40,8 @@ def thredded_anonymous?
       true
     end
 
+    alias_method :anonymous?, :thredded_anonymous?
```

Comment:
Ahhhh yup, I agree with everything you point out. I think this might be a case where we're defining and calling `current_user` from inside thredded - which is usually defined in the parent application. We should probably rename `current_user` to `thredded_current_user`